site/docs/concepts/architecture.md

```diff
@@ -2,27 +2,53 @@
 
 Fybrik takes a modular approach to provide an open platform for controlling
 and securing the use of data across an organization. The figure below showcases the
-current architecture of the Fybrik platform, running on top of Kubernetes. 
+current architecture of the Fybrik platform, running on top of Kubernetes.
 
 The core parts of Fybrik are based on Kubernetes controllers and Custom Resource Definitions (CRDs) in order to define its work items and reconcile the state of them.
 
 The primary interaction object for a data user is the `FybrikApplication` custom resource (1) where a user defines which data should be used for which purpose. The following chart and description describe the architecture and components of Fybrik relative to when they are used.
 
 ![Architecture](../static/architecture.png)
 
-Before the data user can perform any actions a data operator has to [install](../get-started/quickstart.md) Fybrik and modules. 
-[Modules](./modules.md) (M) describe capabilities that can be included in a data plane.  These may be existing open source or third party services, or custom ones.  The module of a service indicates the capabilities it supports, the formats and interfaces, and how to deploy the service.  Modules may describe externally deployed services, or services deployed by fybrik.  Examples of modules are those that provide read/write access or produce implicit copies that serve as lower latency caches of remote assets. Modules may also perform actions to enforce data governance policy decisions, such as masking or redaction as examples.
+## Modules
 
-Fybrik connects to external services to receive data governance decisions, metadata about datasets and credentials. Policies, assets and access credentials to the assets have to be defined before the user can run an application. The current abstraction supports 2 different [connectors](./connectors.md): one for data catalog (2) and one for policy manager (3). It is designed in an open way so that multiple different catalog and policy frameworks of all kinds of cloud and on-prem systems can be supported. The data steward configures policies in an external policy manager over assets defined in an external data catalog. Dataset credentials are retrieved from Vault by using [Vault API](https://www.vaultproject.io/api) (6). Vault uses a custom secret engine implemented with [HashiCorp Vault plugins system](./vault_plugins.md) to retrieve the credentials from where they are stored (data catalog for example).
+Before the data user can perform any actions a data operator has to [install](../get-started/quickstart.md) Fybrik and modules.  
+[Modules](./modules.md) (M) describe capabilities that can be included in a data plane. 
```

Comment:
Please add a link to the Modules section. https://fybrik.io/v1.1/concepts/modules/

Reply:
Sorry for misleading. 
Let's try this way:

"Before the data user can perform any actions, a data operator has to [install](../get-started/quickstart.md) Fybrik and modules; and to configure Fybrik connectors."

Now you start the Modules sub section. 
and into it put 
[Modules](./modules.md) (M) describe capabilities that can be included in a data plane.

Reply:
Done